### PR TITLE
New splash screen for the baklava client

### PR DESCRIPTION
**Pull request by yummyweb**
Please ignore the commits made on and before 20 march.

### CODE REVIEW

File: baklava/splash-screen.html [hunk 1]
```diff
@@ -5,8 +5,20 @@
...
+    <img style="height:140px; position:fixed; top:40%; left:50%; transform:translate(-50%, -50%);" src="./icons/icon.png" alt="DogeHouse Logo">
+    <h3 style="font-family: 'Poppins'; position:fixed; top:60%; left:50%; transform:translate(-50%, -50%);">Fuelling rocket...</h3>
+
+    <script>
+        setTimeout(() => {
+            document.querySelector('h3').textContent = "Reaching space..."
+        }, 1000);
+        setTimeout(() => {
+            document.querySelector('h3').textContent = "Parking on the moon..."
+        }, 2000);
```
Comment (theotterlord) [inline baklava/splash-screen.html]: Would it not be better if the text instead was shown when certain events were triggered. Not sure if this is possible, but would be better than just using a timeout.

File: baklava/src/electron.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ let tray: Tray;
 let menu: Menu;
 let splash;
 
-export const __prod__ = app.isPackaged;
+export const __prod__ = app.isPackaged;
```
Comment (theotterlord) [inline baklava/src/electron.ts]: Fix spacing
```suggestion
export const __prod__ = app.isPackaged;
```

File: kibbeh/src/ui/Tag.tsx [hunk 3]
```diff
@@ -8,6 +8,10 @@ interface TagProps {
 export const Tag: React.FC<TagProps> = ({ children, glow }) => {
   return (
     <div
+<<<<<<< HEAD
+      className={`cursor-pointer bg-primary-700 hover:bg-primary-600 text-xs px-2 font-bold text-primary-100 justify-center items-center rounded`}
+      style={{ height: "22px", boxShadow: "0px 0px 7px rgba(253, 77, 77, 0.3)", borderColor: "#FD4D4D", borderStyle: "solid", borderWidth: 0.2 }}
+=======
```
Comment (theotterlord) [inline kibbeh/src/ui/Tag.tsx]: Merge conflict of some sort?

File: kibbeh/src/ui/Tag.tsx [hunk 4]
```diff
@@ -16,6 +20,7 @@ export const Tag: React.FC<TagProps> = ({ children, glow }) => {
         boxShadow: glow ? "0px 0px 7px var(--color-accent-glow)" : "",
         border: glow ? ".5px solid var(--color-accent)" : "",
       }}
+>>>>>>> efa67978626deb01f424f619ef8abad23e897cdd
```
Comment (theotterlord) [inline kibbeh/src/ui/Tag.tsx]: Same here?

**Review by TheOtterlord — CHANGES_REQUESTED** [review]
As well as the points below, lots of the emotes seem to have been modified for some reason?

File: baklava/splash-screen.html (hunk 1 above)
Comment (yummyweb) [reply baklava/splash-screen.html]: I dont think so that's possible.

Comment (yummyweb) [reply baklava/splash-screen.html]: We could use randomness to like trigger and show these.

File: baklava/src/electron.ts (hunk 2 above)
Comment (yummyweb) [reply baklava/src/electron.ts]: done

File: kibbeh/src/ui/Tag.tsx (hunk 3 above)
Comment (yummyweb) [reply kibbeh/src/ui/Tag.tsx]: ok leme see

File: kibbeh/src/ui/Tag.tsx (hunk 4 above)
Comment (yummyweb) [reply kibbeh/src/ui/Tag.tsx]: done

File: kibbeh/src/ui/Tag.tsx (hunk 3 above)
Comment (yummyweb) [reply kibbeh/src/ui/Tag.tsx]: done

File: .vscode/PythonImportHelper-v2-Completion.json [hunk 5]
```diff
@@ -0,0 +1,2360 @@
+[
```
Comment (amitojsingh366) [inline .vscode/PythonImportHelper-v2-Completion.json]: what is this file for exactly?

File: baklava/splash-screen.html (hunk 1 above)
Comment (amitojsingh366) [reply baklava/splash-screen.html:21]: @yummyweb @TheOtterlord it is very possible. But that would end up increasing the time splash shows

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]
@yummyweb I don't understand why you have deleted so many emojis? Please fix that. Also your `kibbeh/package-lock.json` seems to have over 30k insertions, fix that as well

File: .vscode/PythonImportHelper-v2-Completion.json (hunk 5 above)
Comment (yummyweb) [reply .vscode/PythonImportHelper-v2-Completion.json]: idk, automatically added by vs code, lemme remove it

Comment (yummyweb) [reply .vscode/PythonImportHelper-v2-Completion.json]: done

File: baklava/splash-screen.html (hunk 1 above)
Comment (amitojsingh366) [reply baklava/splash-screen.html]: @yummyweb @TheOtterlord  it is very possible. But that would end up increasing the time splash shows

**Comment by bansalarnav** [comment]
> 
> 
> @yummyweb I don't understand why you have deleted so many emojis? Please fix that. Also your `kibbeh/package-lock.json` seems to have over 30k insertions, fix that as well

`kibbeh/package-lock.json` shouldn't exist as kibbeh has been migrated to yarn

**Comment by yummyweb** [comment]
But i use npm instead of yarn @ArnavBansal11

**Comment by amitojsingh366** [comment]
> 
> 
> But i use npm instead of yarn @ArnavBansal11

You will have to use yarn if it's switched otherwise you might end up causing problems for other contributors.
Also @ArnavBansal11 i don't think it's switched yet as [`kibbeh/package-lock.json`](https://github.com/benawad/dogehouse/blob/staging/kibbeh/package-lock.json) still exists

**Comment by yummyweb** [comment]
i removed kibbeh/package-lock.json

**Comment by amitojsingh366** [comment]
And @yummyweb why are you deleting so many emotes?

**Comment by yummyweb** [comment]
i didnt

**Comment by amitojsingh366** [comment]
> 
> 
> i didnt

Please check the files changed section, it shows you have deleted many emojis

**Comment by yummyweb** [comment]
i know but i didnt delete them

**Comment by amitojsingh366** [comment]
> 
> 
> i know but i didnt delete them

but your if your pull gets merged it will deleted them, so you have to fix it...

**Comment by yummyweb** [comment]
i am working on that

**Comment by bansalarnav** [comment]
> > But i use npm instead of yarn @ArnavBansal11
> 
> You will have to use yarn if it's switched otherwise you might end up causing problems for other contributors.
> Also @ArnavBansal11 i don't think it's switched yet as [`kibbeh/package-lock.json`](https://github.com/benawad/dogehouse/blob/staging/kibbeh/package-lock.json) still exists

I think some PR did it as it was `yarn.lock` till yesterday

**Comment by yummyweb** [comment]
@amitojsingh366 I have added the emotes now, all the emotes.

**Review by HarrisonMayotte — APPROVED** [review]
Looks good to me.

**Review by amitojsingh366 — APPROVED** [review]
Yup looks fine to me now

**Comment by amitojsingh366** [comment]
Tho i feel that 18 commits for something this small were unnecessary

**Comment by HarrisonMayotte** [comment]
That can be fixed when merging though- it can just be merged with "squash and merge" to make it into one big commit rather than 18.

**Comment by amitojsingh366** [comment]
> 
> 
> That can be fixed when merging though- it can just be merged with "squash and merge" to make it into one big commit rather than 18.

Oh? i didn't know that was a thing

**Review by TheOtterlord — APPROVED** [review]

**Comment by amitojsingh366** [comment]
@TheOtterlord @HarrisonMayotte i don't know how to do the squash merge thing and @yummyweb has requested this to be merged. So can you guys handle that?

**Pull request merged**